waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -115,19 +120,21 @@
       [[spec
         [:metadata name namespace uid [:annotations waiter/service-id]]
         [:status {replicas 0} {availableReplicas 0} {readyReplicas 0} {unavailableReplicas 0}]] replicaset-json
+       replicaset-version (get-in replicaset-json [:metadata :labels :waiter/replicaset-version] nil)
```

Comment:
Since we expect this to be a timestamp, we should probably make that explicit in the name... E.g., `:waiter/revision-timestamp`, or `:waiter/service-patch-timestamp`.

Reply:
Changed the name to `revision-timestamp`